pom.xml

```diff
@@ -108,40 +108,12 @@
 
     <dependencyManagement>
         <dependencies>
-            <!-- project dependencies - runtime and compile -->
             <dependency>
                 <groupId>io.kroxylicious</groupId>
-                <artifactId>kroxylicious</artifactId>
+                <artifactId>kroxylicious-bom</artifactId>
                 <version>${project.version}</version>
```

Comment:
Is there a cyclic dependency here at release time? If we tried to release the bom first then the other dependencies wouldn't be released yet. If we tried to release the other components first then they would be referencing a snapshotted bom.xml.

Reply:
Not sure how to work around that one, it starts looking like an additional artifact we offer alongside the rest, not something that we would be able to use internally. I guess that's how jackson do it, boms have their own repo https://github.com/FasterXML/jackson-bom

Reply:
I haven't tried to run a release yet, thats a good point to test, but I think maven works it out and just sorts the bom first in the reactor (because the bom doesn't use the parent pom)

Reply:
Yeah testing the release process seems to work without changes and produces https://github.com/kroxylicious/kroxylicious/pull/491/files

Reply:
Hmm actually the build for https://github.com/kroxylicious/kroxylicious/pull/491/files failed with a 0.3.0-SNAPSHOT reference so something is still missing.

Reply:
Is the BOM change in that PR? I can't see `kroxylicious-bom` in it's tree

Reply:
Ohhhhhhhh your right I forgot to set the sets to base it on the bom branch